src/loader/Preprocessor.py

```diff
@@ -122,3 +119,22 @@ def _normalize_standardscaler(
                 transformed_array, columns=recording.sensor_frame.columns
             )
         return recordings
+    
+    def _normalize_minmaxscaler(self, recordings: "list[Recording]") -> "list[Recording]":
+        """
+        Normalizes the sensor values to be in range 0 to 1
+        """
+        assert_type([(recordings[0], Recording)])
+
+        # First fit the scaler on all data
+        scaler = MinMaxScaler()
+        for recording in recordings:
+            scaler.fit(recording.sensor_frame)
+
+        # Then apply normalization on each recording_frame
+        for recording in recordings:
+            transformed_array = scaler.transform(recording.sensor_frame)
+            recording.sensor_frame = pd.DataFrame(
+                transformed_array, columns=recording.sensor_frame.columns
+            )
+        return recordings
```

Comment:
This pretty much duplicates `_normalize_standardscaler`

Solution: Let the normalize function take a scaler that is applied
```python
_normalize(self, recordings: list[Recording], scaler_fn=StandardScaler):
        """
        Normalizes the sensor values with the given scaler
        """
        assert_type([(recordings[0], Recording)])

        # First fit the scaler on all data
        scaler = scaler_fn()
        for recording in recordings:
            scaler.fit(recording.sensor_frame)

        # Then apply normalization on each recording_frame
        for recording in recordings:
            transformed_array = scaler.transform(recording.sensor_frame)
            recording.sensor_frame = pd.DataFrame(
                transformed_array, columns=recording.sensor_frame.columns
            )
        return recordings


def _normalize_minmaxscaler(self, recordings: "list[Recording]") -> "list[Recording]":
    return self._normalize(recordings, scaler=MinMaxScaler)

def _normalize_standardscaler(self, recordings: "list[Recording]") -> "list[Recording]":
    return self._normalize(recordings)
```